### ISSUE TITLE
Weekly Top 10 Dataset

### ISSUE THREAD

**Issue by jerryrou** [Anoopkjomi/Netflix-Movies-and-TV-Shows-Analysis#1]
Hi - 
I came across your Netflix project- nice work.  I was wondering if you know where I can find the following data.
Basically, I am looking for years 2021 and 2022 for each week, the list of top 10 films and TV shows and the number of hours viewed)

Dataset variables

- week	
- category (Film English; Film Non-English; TV English; TV Non-English)	
- weekly_rank (1-10)	
- show_title	
- season_title	
- Sum(weekly_hours_viewed)

Any assistance would be appreciated - thanks - Jerry

![Netflix-Example](https://user-images.githubusercontent.com/80755902/206849139-fc737ae1-4c7a-481f-8a4d-3c15c862be24.JPG)

**Comment by Anoopkjomi** [comment]
May I know who you are

**Comment by jerryrou** [comment]
Yes I am a college professor looking for this data to create a project for my students (data analysis project).

**Comment by Anoopkjomi** [comment]
Ok Sir I will  do my best

**Comment by jerryrou** [comment]
Thank you - please let me know what you come up with - Jerry
